eng/tox/verify_whl.py

```diff
@@ -107,16 +106,33 @@ def should_verify_package(package_name):
 
     # get target package name from target package path
     pkg_dir = os.path.abspath(args.target_package)
-    pkg_name, namespace, ver = get_package_details(os.path.join(pkg_dir, "setup.py"))
+    pkg_name, namespace, ver, python_requires = get_package_details(os.path.join(pkg_dir, "setup.py"))
+
+    top_level_module = namespace.split(".")[0]
+    wheel_location = get_wheel(args.dist_dir, ver)
 
-    top_level_module = namespace.split('.')[0]
+    if not os.getenv("IGNORE_WHEEL_PYVERSION_CHECK"):
+        if not python_requires:
+            logging.info("Your package must define a setup argument for 'python_requires'.")
+            exit(1)
+        else:
+            if not parse_version(trim_spec(python_requires)) >= parse_version("3.0"):
+                logging.info(
+                    "The python_requires value of '{}' should instead be at least '>=3.0'.".format(python_requires)
```

Comment:
Could we make it to at least 3.6?

Reply:
No problem! I was expecting points of clarity like this 👍

Reply:
Can you store the `"3.0"` into a global variable, this way when we do 3.6, 3.7, etc. it's one update in one place? Something like
```python
MIN_PY_VERSION="3.6"

if not parse_version(trim_spec(python_requires)) >= parse_version(MIN_PY_VERSION):
```